### PR TITLE
Migrate submodule

### PR DESCRIPTION
**Pull request by JernKunpittaya**
-We believe our onnx2circom lib that consists of onnx2keras and keras2circom will diverge more from their original, so we change from submodule approach to have all code here instead for optimizing in future
-Change some codes abt folder stuffs
-Update ReadMe

### CODE REVIEW

File: tests/onnx2circom/test_onnx_to_circom.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 
 # NOTE: Change the path to your own path
-CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/path/to/circom-2-arithc-project-root')
-MP_SPDZ_PROJECT_ROOT = Path('/path/to/mp-spdz-project-root')
+CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/Users/jernkun/circom-2-arithc')
```
Comment (mhchia) [inline tests/onnx2circom/test_onnx_to_circom.py:17]: should we changed these back to 

```python
CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/path/to/circom-2-arithc-project-root')
MP_SPDZ_PROJECT_ROOT = Path('/path/to/mp-spdz-project-root')
```
or we can add a `tests/onnx2circom/.env` to store the paths, and load them in the tests with [python-dotenv](https://pypi.org/project/python-dotenv/)

File: zkstats/onnx2circom/README.md
Comment (mhchia) [inline zkstats/onnx2circom/README.md:1]: It seems this content is identical to `tests/onnx2circom/README.md`. How about we just remove the sections related to submodules `zkstats/onnx2circom/README.md` and leave the relevant sections `## Run onnx2circom` and `## Import`? And we can also add a reference section for the repos we forked from.

IMO these sections will still be helpful for people who want to learn more about and use this onnx2circom package. What do you think?

**Review by mhchia — COMMENTED** [review]
Great work 🙏 I've left some comments

File: tests/onnx2circom/test_onnx_to_circom.py (hunk 1 above)
Comment (JernKunpittaya) [reply tests/onnx2circom/test_onnx_to_circom.py:17]: Thanks!! Resolved :))

File: zkstats/onnx2circom/README.md
Comment (JernKunpittaya) [reply zkstats/onnx2circom/README.md:1]: Agree, I will move this content to tests/onnx2circom/README.md instead. For this file zkstats/onnx2circom/README.md will contain Run onnx2circom and Impoart :)))

**Pull request merged**